src/redux/actions/posts.js

```diff
@@ -41,3 +42,7 @@ export const updatePostLink = newPost => {
 export const updatePostTitle = newPost => {
   return { type: 'UPDATE_POST_TITLE', payload: newPost }
 }
+
+export const newPostPage = () => {
```

Comment:
this shouldn't be an action, because you're not actually dispatching anything. So it's nothing todo with redux / actions.